dockerfiles/mlrun/requirements.txt

```diff
@@ -8,3 +8,5 @@ mpi4py~=3.1
 pymysql~=1.1
 # consumed when training a model using PyTorch
 tqdm~=4.66
+sqlalchemy~=2.0
+sqlalchemy-utils~=0.41.2
```

Comment:
why is it needed on user's dockerfile? mlrun/mlrun is aimed to runs jobs - not to run db thingies

Reply:
lets create a follow-up pr moving `mlrun/common/db/sql_session.py` to server side and remove these requirements from mlrun/mlrun image